cmd/crc/cmd/start.go

```diff
@@ -18,6 +18,7 @@ func init() {
 
 	startCmd.Flags().StringVarP(&bundlePath, "Bundle", "b", constants.DefaultBundle, "The system bundle used for deployment of the OpenShift cluster.")
 	startCmd.Flags().StringP(config.VMDriver.Name, "d", constants.DefaultVMDriver, fmt.Sprintf("The driver to use for the CRC VM. Possible values: %v", constants.SupportedVMDrivers))
+	crcConfig.BindFlag(config.VMDriver.Name, startCmd.Flags().Lookup("vm-driver"))
```

Comment:
This means we are going to do that for each flag which we take from the config side, it is not scaleable, I would like to have it like we had in minishift.

Reply:
Agree

Reply:
However, this is simplistic for now and works... right?

Reply:
We do not have any other flags now, so i didn't create a wrapper around the `BindPFlags` where we can pass in a flagset!